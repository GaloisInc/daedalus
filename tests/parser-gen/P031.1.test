P031.ddl
--gen
--interp=inputs/P031.1.inp
