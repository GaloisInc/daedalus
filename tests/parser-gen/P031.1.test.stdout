--- Found 1 results:
{ res: { x: "aaaaa"
       , y: "ccccc"
       , z: "ccccc"
       }
}
